charts/fybrik/files/taxonomy/fybrik_module.json

```diff
@@ -0,0 +1,70 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "$id": "fybrik_module.json",
+    "title": "FybrikModule taxonomy",
+    "type": "object",
+    "description": "Values contributed to by Fybrik modules.",
+    "properties": {
+        "spec": {
+            "type": "object",
+            "properties": {
+                "pluginType": {
+                    "$ref": "taxonomy.json#/definitions/PluginType"
+                },
+                "capabilities": {
+                    "type": "array",
+                    "items": {
+                        "type": "object",
+                        "properties": {
+		                    "capability": {
+                                "$ref": "taxonomy.json#/definitions/Capability"
+                            },
+                            "api": {
+                                "type": "object",
+                                "properties": {
+                                    "interfaceDetails": {
```

Comment:
There is no such property in `api`. Please see https://fybrik.io/dev/reference/crds/#fybrikmodulespeccapabilitiesindexapi. 

That is `api` itself is a variant of Interface that also includes endpoint as a required field.